_data/students.yml

```diff
@@ -0,0 +1,26 @@
+# Please add your entries to a random position in the file.
+# This is to remove conflicts with other people adding entries.
+
+- name: Miguel Piedrafita
+  github: m1guelpf
+  image: m1guelpf.png
+  blog: https://miguelpiedrafita.com/
+  twitter: m1guelpf
+
+- name: Chahat Bansal
+  github: chahatbansal
+  image: chahat.png
+  blog:
+  twitter:
```

Comment:
Don't include twitter and blog if they are not provided. It will lead to a condition being evaluated to true in index.html